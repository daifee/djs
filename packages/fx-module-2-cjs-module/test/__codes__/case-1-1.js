
fxDefine('case-1', function () {
  function fly() {
    //
  }

  const ID = 'daif'

  return {
    fly,
    age: 18,
    id: ID
  };
});
