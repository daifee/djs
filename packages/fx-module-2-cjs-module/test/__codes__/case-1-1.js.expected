"use strict";

function fly() {
  //
}
const ID = 'daif';
module.exports = {
  fly,
  age: 18,
  id: ID
}